src/pkg/cover_crypt/.gitignore

```diff
@@ -1,5 +1 @@
-*.wasm*
-*.js
-*.json
-*.ts
-*.md
+*
```

Comment:
there is a `.gitkeep` file in this folder (that will be ignored with the wildcard `*`), we should make sure this empty folder is not removed

Reply:
The `.gitkeep` will not be removed if it already exists in Git when the `.gitignore` is added (or if you force add it after the `.gitignore`)